### PR TITLE
security: extend overview to cover secure boot

### PR DESCRIPTION
**Pull request by ricardosalveti**
Signed-off-by: Ricardo Salveti <ricardo@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 2a9dd2fc3d80c87cddcce63c26d11e60c313a521 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1276/docs/artifacts/html/index.html

**Review by doanac — APPROVED** [review]
feels good to see progress here.

File: source/reference-manual/security/security.rst [hunk 1]
```diff
@@ -42,10 +43,16 @@ in order to generate :ref:`production targets <ref-production-targets>`.
 .. _targets.json:
    https://theupdateframework.com/metadata/
 
-Secure Boot
------------
+Secure Boot (Hardware Root of Trust)
+------------------------------------
 
-TODO
+:ref:`Secure Boot <ref-secure-boot>` is the mechanism used to force a device to only execute boot software
```
Comment (vanmaegima) [inline source/reference-manual/security/security.rst:49]: maybe keep this under 80 columns (we might need to fix this in several parts of the docs)

File: source/reference-manual/security/security.rst [hunk 2]
```diff
@@ -42,10 +43,16 @@ in order to generate :ref:`production targets <ref-production-targets>`.
...
+that is signed by a certain set of keys. The verification process and respective
+security functions is performed by the SoC boot ROM, and it is the starting point
```
Comment (vanmaegima) [inline source/reference-manual/security/security.rst:51]:
```suggestion
security functions are performed by the SoC boot ROM, and these are the starting points
```

File: source/reference-manual/security/security.rst (hunk 1 above)
Comment (ricardosalveti) [reply source/reference-manual/security/security.rst:49]: Yeah, was curious if the rendering would have any effect here but it doesn't break the line, let me fix it (and we should probably have a linter for this (@kprosise).

**Comment by doanac** [comment]
Docs for 9bfd9b30bae6ea7490a758f43fcffe90a838f3f4 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1279/docs/artifacts/html/index.html

**Comment by ricardosalveti** [comment]
@vanmaegima updated.

**Comment by doanac** [comment]
Docs for e03d2a45eacb9c475319de778c37526975a90021 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1280/docs/artifacts/html/index.html

**Review by MrCry0 — APPROVED** [review]
LGTM

**Review by angolini — APPROVED** [review]
LGTM

**Pull request merged**